sdk/appcenter/src/test/java/com/microsoft/appcenter/utils/context/AuthTokenContextTest.java

```diff
@@ -109,7 +110,9 @@ public void setAuthTokenTest() {
 
         /* Verify that listener is called. */
         verify(mockListener, times(2)).onNewAuthToken(notNull(String.class));
-        verify(mockListener, times(1)).onNewUser(notNull(String.class));
+        ArgumentCaptor<UserInformation> captorArg = ArgumentCaptor.forClass(UserInformation.class);
+        verify(mockListener, times(1)).onNewUser(captorArg.capture());
```

Comment:
`times(1)` is always assumed unless specified.

```suggestion
        verify(mockListener).onNewUser(captorArg.capture());
```